test/test_transactions.rb

```diff
@@ -40,6 +40,8 @@ def get_transactions_with_retries(access_token, start_date, end_date, count: nil
   end
 
   def test_get
+    skip 'skipping flaky test'
```

Comment:
I filed [CLIB-283075](https://jira.plaid.com/browse/CLIB-283075) to track the flaky test issue.